include/quda.h

```diff
@@ -1701,6 +1708,21 @@ extern "C" {
 
   void setMPICommHandleQuda(void *mycomm);
 
+  /**
+   * Performs two-link Gaussian smearing on a given spinor (for staggered fermions).
+   * @param h_in        Input spinor field to smear
+   * @param inv_param   Contains all metadata regarding host and device storage 
+   *                    and operator which will be applied to the spinor
+   * @param n_steps     Number of steps to apply
+   * @param width       The width of the Gaussian
+   * @param compute_2link nonzero to compute two-link,
+   *                      zero to reuse gaugeSmeared (default = 1)
+   * @param delete_2link nonzero to delete two-link after smearing,
+   *                     zero to keep two-link for future use (default = 0)
+   * @param t0          Set if the input spinor is on a time slice (default = -1)
+   */
+  void performTwoLinkGaussianSmearNStep(void *h_in, QudaInvertParam *inv_param, const int n_steps, const double width, const int compute_2link /*= 1*/, const int delete_2link /*= 0*/, const int t0 /*= -1*/);
```

Comment:
Add missing `[in]` / `[out]` labels to this function.

Reply:
The doxygen mentions that the parameters have default values, but the signature shows these are not present.  Indeed, default parameters are not allowed with regular C.

Reply:
as per our previous discussions, all params are now grouped in a new structure

Reply:
This looks much cleaner now 😄